justfiles/backend.just

```diff
@@ -1,11 +1,15 @@
+SUDO := if `groups | grep -q docker > /dev/null 2>&1 && echo true || echo false` == "true" { "" } else { "sudo" }
+PWD := `pwd`
+WAVS_CMD := "docker run --rm --network host -e WAVS_CLI_ETH_MNEMONIC=\"test test test test test test test test test test test junk\" -e WAVS_SUBMISSION_MNEMONIC=\"test test test test test test test test test test test junk\" -v $(pwd):/data -v $(pwd)/cli:/data/.wavs-data/cli ghcr.io/lay3rlabs/wavs:0.3.0-beta"
```

Comment:
could also put in a .env in the `/data/config` ? but the .env should be committed since it's an example. not sure how this worked before unless it was not required back in alpha5